scipy/linalg/flapack_gen_banded.pyf.src

```diff
@@ -122,11 +122,45 @@ subroutine <prefix2c>gbcon(norm,n,kl,ku,ab,ldab,ipiv,anorm,rcond,work,rwork,info
     integer intent(in),check(ku>=0) :: ku
     <ftype2c> dimension(ldab,n),intent(in) :: ab
     integer optional,intent(in),depend(ab),check(ldab >= (2*kl+ku+1)),depend(ab,kl,ku) :: ldab=2*kl+ku+1
-    
     integer intent(in), depend(n), dimension(n) :: ipiv
     <ftype2> intent(in) :: anorm
     <ftype2> intent(out) :: rcond
     <ftype2c> intent(hide, cache), dimension(2*n), depend(n) :: work
     <ftype2> intent(hide, cache), dimension(n), depend(n) :: rwork
     integer intent(out) :: info
 end subroutine <prefix2c>gbcon
+
+
+function <prefix>langb(norm,n,kl,ku,ab,ldab,work)
```

Comment:
It does not change much but better to be consistent with f2py conventions.

```suggestion
function <prefix>langb(norm,n,kl,ku,ab,ldab,work) result(n2)
```

Other than that looks good to me.